engine/render/src/wscript

```diff
@@ -16,15 +16,27 @@ def build(bld):
     else:
         sources += ['platform/renderer_common.cpp']
 
-    obj = bld.stlib(features        = 'cxx ddf',
-                    includes        = ['.', '../build/proto'],
-                    proto_gen_py    = True,
-                    protoc_includes = '../proto',
-                    source          = sources,
-                    target          = 'render')
+    # remove the font render plugins
+    sources = [x for x in sources if not str(x).endswith('font_renderer_default.cpp')]
+    #sources = [x for x in sources if not str(x).endswith('font_renderer_raqm.cpp')]
```

Comment:
what is font_renderer_raqm.cpp?

Reply:
It's a suggested name for the other backend we'll implement. "raqm" is the lib used there (which calls harfbuzz and sheenbidi etc)